service/isiService.go

```diff
@@ -477,21 +540,41 @@ func (svc *isiService) AddExportClientNetworkIdentifierByIDWithZone(ctx context.
 		return err
 	}
 
-	log.Debugf("AddExportClientNetworkIdentifierByID client FQDN '%s' client IP '%s'", clientFQDN, clientIP)
+	currentClient, err := getClientToUseForNodeID(ctx, clusterToNodeIDMap, clusterName, nodeID)
```

Comment:
since the clusterToNodeIDMap is package variable, we need not pass it. not necessary to change it.

Reply:
agree with you, but I went for this signature considering making this available for other maps also if we introduce them in the future. Otherwise, it will be tied to only this variable.

Reply:
Okay.